dora/src/stdlib.rs

```diff
@@ -117,7 +117,7 @@ pub extern "C" fn throw_native(val: bool) {
     if val {
         handle_scope(|| {
             let vm = get_vm();
-            let obj = alloc_exception(vm, Ref::null());
+            let obj = alloc_exception(vm, Str::empty(vm));
```

Comment:
I thought this would resolve the issue, but it didn't. Did I miss anything here?